README.md

```diff
@@ -359,7 +358,7 @@ The `per_sample_metrics.tsv` file produces a row per sample.
 
 The `per_project_metrics.tsv` file aggregates the metrics by project (aggregates the metrics across
 samples with the same project) and has the same columns as [per_sample_metrics.tsv](#per_sample_metricstsv).
-In this case, `barcode_name` and `library_name` will contain the project name (or `None` if no
+In this case, `sample_ID` and `library_name` will contain the project name (or `None` if no
```

Comment:
Can you search for `library_name` here and through-out the code to ensure we've removed it?

Reply:
Fixed now.